clj-refactor.el

```diff
@@ -786,6 +793,10 @@ A new record is created to define this constructor."
      car)
    ""))
 
+(defun cljr--inside-project-p ()
```

Comment:
Think this should be called `cljr--inject-middleware-p?` to better reflect its usage.   

Also, shouldn't it have an argument list like `(&rest _)` since it's actually expecting 3 arguments, that are ignored?

Edit: I see it's used in two places, so I'd suggest leaving this and creating another function, `cljr--inject-middleware-p` that just has the right argument list and calls out to this one.